File: algorithm/Stack/Stack.java
import java.util.*;

public class stack {
    public static void main(String[] args) {
        Stack<Integer> stack = new Stack<>();

        stack.push(1);
        stack.push(2);
        stack.push(3);
        System.out.println("현재 스택: " + stack);

        System.out.println("pop: " + stack.pop());
        System.out.println("pop: " + stack.pop());
        System.out.println("남은 스택: " + stack);
    }
}

File: algorithm/Stack/Stack.py
# Stack (스택)
# LIFO (후입선출) 구조
# append()로 push, pop()으로 pop

stack = []

stack.append(1)
stack.append(2)
stack.append(3)
print("현재 스택:", stack)

print("pop:", stack.pop())
print("pop:", stack.pop())
print("남은 스택:", stack)
